static/js/subs.js

```diff
@@ -698,14 +702,7 @@ $(function () {
             return;
         }
```

Comment:
Just to be clear, I've to show the input outside of modal if `should_list_all_streams` is false, right?